### ISSUE TITLE
Release Reward-Forcing-T2V-14B on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [JaydenLyh/Reward-Forcing#2]
Hi @JaydenLyh 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your excellent work, "Reward Forcing: Efficient Streaming Video Generation with Rewarded Distribution Matching Distillation", through Hugging Face's daily papers. The paper page is here: https://huggingface.co/papers/2512.04678.
The paper page lets people discuss your paper and find related artifacts (like your models and code). You can also claim the paper as yours, which will show up on your public profile at HF, and add GitHub and project page URLs.

It's fantastic to see that your `Reward-Forcing-T2V-1.3B` model checkpoint is already available on the Hugging Face Hub, improving its discoverability and usage!

I also noticed from your GitHub README that a `Reward-Forcing-T2V-14B` model is currently "in progress". When this larger model is ready for release, would you be interested in hosting it on https://huggingface.co/models as well?

Hosting additional models on Hugging Face will provide even more visibility and enable better discoverability. We can add tags to the model cards so that people can easily find your models and link them to the paper page.

If you're interested, here's a guide: https://huggingface.co/docs/hub/models-uploading. For custom PyTorch models, the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class simplifies adding `from_pretrained` and `push_to_hub` functionalities.

You can also build a demo for your model on [Spaces](https://huggingface.co/spaces), and we can provide you a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you A100 GPUs for free.

Let me know if you're interested or need any guidance when the `Reward-Forcing-T2V-14B` model is ready!

Kind regards,

Niels
ML Engineer @ HF 🤗

**Comment by JaydenLyh** [comment]
Thank you and the Hugging Face team for your interest in our work. We are currently fully engaged in training the 14B model, but it will take some time. We will release it on Hugging Face once the training is complete.
